src/Mod/Fem/femobjects/constraint_electrostaticpotential.py

```diff
@@ -30,7 +30,7 @@
 #  \ingroup FEM
 #  \brief constraint electrostatic potential object
 
-from FreeCAD import Units
+# from FreeCAD import Units
```

Comment:
This seems wrong. App::PropertyElectricPotential uses a unit as far as I see at a quick look.

Reply:
But we don't use `App::PropertyElectricPotential` in the script?